modules/loramac-node/CMakeLists.txt

```diff
@@ -0,0 +1,55 @@
+if(${CONFIG_HAS_SEMTECH_RADIO_DRIVERS})
+
+set(ZEPHYR_CURRENT_LIBRARY loramac-node)
+
+zephyr_library_include_directories(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/boards)
+zephyr_library_include_directories(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/system)
+zephyr_library_include_directories(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/radio)
+
+zephyr_library_sources(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/boards/mcu/utilities.c)
+
+zephyr_library_sources(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/system/timer.c)
+zephyr_library_sources(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/system/delay.c)
+zephyr_library_sources(${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/system/systime.c)
+
+zephyr_library_sources_ifdef(CONFIG_HAS_SEMTECH_SX1272 ${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/radio/sx1272/sx1272.c)
```

Comment:
There are some very long lines here and below (>100 chars long).

Could you please wrap those long lines, for example as:
```
zephyr_library_sources_ifdef(CONFIG_HAS_SEMTECH_SX1272
  ${ZEPHYR_LORAMAC_NODE_MODULE_DIR}/src/radio/sx1272/sx1272.c
)
```